.travis.yml

```diff
@@ -2,6 +2,7 @@ sudo: false
 language: python
 python:
   - "2.7"
+  - "3.4"
```

Comment:
Python 3.5 lead to strange package version issues with conda, I hence opted for 3.4.